pkg/sosreport/run-sosreport.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+if grep -q 'CPE_NAME=.*atomic-host' /etc/os-release; then
```

Comment:
Fedora also the atomic command, but we want to run sosreport directly there.  So we could check `atomic && !sosreport`.  But then we might use the wrong image... so I thought it is best to just detect rhel atomic directly....  What do you think?